File: test/ocamlTests/CharTest/isUppercaseTest.ml
test "isUppercase('A')" (fun () -> expect (Char.isUppercase 'A') |> toEqual Eq.bool true) ; 
test "isUppercase('7')" (fun () -> expect (Char.isUppercase '7') |> toEqual Eq.bool false) ; 
test "isUppercase('\237')" (fun () -> expect (Char.isUppercase '\237') |> toEqual Eq.bool false) ; 

File: test/fsharpTests/CharTest/isUppercaseTest.fs
testCase "isUppercase('A')" 
<| fun _ -> 
    let expected = true
    Expect.equal expected (Char.isUppercase 'A') "error"
testCase "isUppercase('7')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isUppercase '7') "error"
testCase "isUppercase('\237')" 
<| fun _ -> 
    let expected = false
    Expect.equal expected (Char.isUppercase '\237') "error"
